pkg/apis/workload/v1alpha1/types.go

```diff
@@ -110,6 +122,20 @@ const (
 	// The format for the value of this annotation is: JSON Patch (https://tools.ietf.org/html/rfc6902).
 	ClusterSpecDiffAnnotationPrefix = "experimental.spec-diff.workload.kcp.dev/"
 
+	// SummarizingRulesAnnotation
+	//
+	//   experimental.summarizing.workload.kcp.dev
+	//
+	// on upstream resources storing the JSON-encoded summarizing rules for this instance of the resource.
+	// The drives what fields should be overridden in the syncer view, and available for summarizing,
+	// and how they should be managed.
+	//
+	// Typical value is:
+	//    [{"path": ["status"], "promoteToUpstream": true}]
```

Comment:
what does "typical" mean? Are there other fields?

Reply:
Yes, there could be other fields reported by the Syncer and maintained in the SyncerView, even from the Spec, for example `Service.Spec.ClusterIP`

Reply:
I don't mean fields of the payload. Talking about fields of this `[{"path": ["status"], "promoteToUpstream": true}]` thing. What is its schema? Where is it defined? It's an API. We must be precise.

Reply:
It's experimental, but I'll further document it.

Reply:
I added documentation.